### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by AlexZigante** [Mortennn/Dozer#170]
Heyy, is this project dead?

**Comment by TANG617** [comment]
It's still usable, but it may be out of support.(unofficial)

**Comment by Lundeful** [comment]
Looks pretty dead, but if it works it works. Someone might fork it and fix some stuff though

**Comment by chkpnt** [comment]
I'm using https://github.com/dwarvesf/hidden nowadays.

**Comment by gingerbeardman** [comment]
it still works.

Hidden is close, but missing some Dozer things.

**Comment by LinnJS** [comment]
Looking pretty dead I was hoping to see some support here for the new ARM architecture.

**Comment by gingerbeardman** [comment]
> Looking pretty dead I was hoping to see some support here for the new ARM architecture.

You didn't look hard enough @LinnJS : https://github.com/Mortennn/Dozer/pull/154

Working well on my ARM MBP. I use it when I have my external display connected. When I am on the laptop Bartender is better because it deals with the notch.

**Comment by LinnJS** [comment]
@gingerbeardman Thanks for the follow up, so what led me here is in my Activity Monitor it said that this applet was on `intel` architecture rather than `silicon` it works on my M1 but I had to install rosetta to get it to run rather than it running natively.

**Comment by gingerbeardman** [comment]
I don't have Rosetta installed and the ARM build I linked runs  well.

**Comment by BlackBird11** [comment]
for me it does not work on my M1 Max with MacOS 12.6 .. there is no arrow in the menu bar

**Comment by gingerbeardman** [comment]
@BlackBird11 can you say more? There should be dots in the menu bar, not arrows.

I use it [ARM version](https://github.com/Mortennn/Dozer/pull/154) on an M1 Pro running 12.6.

**Comment by yndajas** [comment]
Running the linked build on a 2022 M2 Air, I just get one icon

<img width="633" alt="image" src="https://user-images.githubusercontent.com/40244233/210839189-71793746-4119-48f9-ac10-eed28c1093b5.png">

**Comment by gingerbeardman** [comment]
> Running the linked build on a 2022 M2 Air, I just get one icon

Correct.  This is as expected!

- Click the dot 1 to reveal the dot 2.
- Option-click the single dot 1 to reveal dots 2 and 3.

[See README.md](https://github.com/Mortennn/Dozer/blob/master/README.md#%EF%B8%8F-dozer-icons) for descriptions of the different dots.

**Comment by robrecord** [comment]
Fork here https://github.com/Schroedi/Dozer

**Comment by gingerbeardman** [comment]
> Fork here

Would be good to know the benefits of your fork. Cheers! @robrecord 

Also, this original project is still accepting PRs and is _not_ discontinued.

**Comment by robrecord** [comment]
> ...this original project is still accepting PRs and is _not_ discontinued.

My apologies for the drive-by link @gingerbeardman, and this is brilliant to know the project is alive. 

Someone had asked of the existence of a fork above, and having discovered a live one, thought I'd share.

> Would be good to know the benefits of your fork.

I have not tried the fork yet, but I see it has several commits since the last commit of this parent repo. Honestly I would rather stay with the original as presumably the prebuilt binaries will be posted in the release area; not the case with the fork.

**Comment by gingerbeardman** [comment]
I checked out the fork and the only changes seem to be: changes for building/running on Sonoma, removing the auto-update mechanism (because unofficial fork), and removal of mentions of the downloads (because unofficial fork). So, nothing that doesn't already exist in pull requests here and worse in a number of ways.

I will try to get in contact with Mortenn to get a new build out.

**Comment by robrecord** [comment]
> I checked out the fork and the only changes seem to be: changes for building/running on Sonoma, removing the auto-update mechanism (because unofficial fork), and removal of mentions of the downloads (because unofficial fork). So, nothing that doesn't already exist in pull requests here and worse in a number of ways.

Thank you for evaluating, much appreciated. 

> I will try to get in contact with Mortenn to get a new build out.

It would be great to see those PRs in a new build.

**Comment by fuzzy76** [comment]
> Also, this original project is still accepting PRs and is _not_ discontinued.

A project that haven't released a build for over 4 years is dead by my standards. But if you could get it back on track, that would be great.

**Comment by gingerbeardman** [comment]
I _hate_ that view of software releases.

Old !== dead. PRs have been merged many times since the last release. And you can build it yourself, or use one of the community provided builds, or just carry on using the existing build which still works for many people.

If it was dead, the project would be put into archived status.

**Comment by fuzzy76** [comment]
No. When a projects is abandoned, there usually isn't anyone left to put it into archived status. Github is _full_ of examples. And when you release prebuilt software, a large part of your user mass are end-users that do not have the know-how to compile software. From a non-developer standpoint, lack of releases is death.

**Comment by gingerbeardman** [comment]
Don't be ridiculous. I use tons of software that haven't had releases in aeons. Are they dead? No. I still use them.

I believe only the developer of the project can declare it dead, because it would be their intent. Everything else is just opinion.

But were deep in the semantical weeds here. You say tomato I say tomato. 

With best wishes for a Merry Christmas.

**Comment by fuzzy76** [comment]
Today the cask was completely removed from homebrew. It's 4,5 years since the last pull request with code changes was accepted.

**Comment by gingerbeardman** [comment]
Yet ...it remains usable for many people on many versions of macOS. Of course that's not everybody, but such is life.
